slack-clipper-backend/app.py

```diff
@@ -36,38 +37,50 @@ def get_conversation_thread(client, channel, ts):
         messages.append({
             "text": message["text"],
             "name": users[user_id]["name"],
-            "profile_picture": users[user_id]["profile_picture"],
-            "files": message["files"]
+            "profile_picture": users[user_id]["profile_picture"]
         })
 
+        if "files" in message:
+            permalinksToAttachments = ""
+            for file in message["files"]:
+                if not file['public_url_shared']:
+                    app.client.files_sharedPublicURL(file=file["id"], token=os.environ.get("USER_TOKEN"))
+                permalinksToAttachments += '<a href="'+file["permalink_public"]+'">Download</a><div>'
+            messages.append({
+                "text": "<i>Attachments:</i> \n"+permalinksToAttachments,
+                "name": users[user_id]["name"],
+                "profile_picture": users[user_id]["profile_picture"]
+            })
+
     return messages
 
 
-def make_markdown_message(name, profile_picture, text, files):
-    returnString = f"""
+def make_markdown_message(name, profile_picture, text):
+
+    while "```" in text:
+        text=text.replace("```", "<code>", 1)
+        text = text.replace("```", "</code>", 1)
+    text=text.replace("\n", "<div>")
```

Comment:
you don't need to replace some new lines with the closing div tag?

Reply:
why are we replacing the markdown code backticks? I think they should render properly in the .md file, no?

Reply:
just tested it, the backticks don't render the code properly for some reason in both markdown viewers I tried.

Reply:
can you send me the markdown file?